presto-session-property-managers/src/main/java/com/facebook/presto/session/FileSessionPropertyManager.java

```diff
@@ -46,11 +48,13 @@
             .listJsonCodec(SessionMatchSpec.class);
 
     private final List<SessionMatchSpec> sessionMatchSpecs;
+    private final PrestoVersion coordinatorVersion;
```

Comment:
Why is it called `coordinatorVersion`? Does it only represent coordinator version or represents the cluster version?

Reply:
Yeah, I can change it to prestoVersion